server/config/sqlize.js

```diff
@@ -165,14 +165,8 @@ UserMapRank.belongsTo(User, { foreignKey: 'userID'});
 UserMapRank.belongsTo(Map, { foreignKey: 'mapID'});
 
 if (env !== 'test') {
-	forceSyncDB().then(() => {
-		// Create our default XP systems table if we don't already have it
-		return xpSystems.initXPSystems(XPSystems);
-	}).then(() => {
-		console.log(`Database & tables created!`)
-	}).catch(err => {
-		console.error(err);
-	});
+    // Create our default XP systems table if we don't already have it
```

Comment:
Mix of spaces and tabs again.

Perhaps we should look into getting a linter, soon.

Reply:
You can edit this and force-push it to be right. No need to pollute the history over this.

Reply:
Dang haha x) Yeah, a linter would probably be nice. Will do a force-push this time